dll/win32/netcfgx/lang/ro-RO.rc

```diff
@@ -1,46 +1,52 @@
-    CONTROL "&Obține automat adresă IP", IDC_USEDHCP, "BUTTON", BS_AUTORADIOBUTTON | WS_GROUP | WS_TABSTOP, 14, 43, 210, 12
-    CONTROL "Folosește adresa ur&mătoare:", IDC_NODHCP, "BUTTON", BS_AUTORADIOBUTTON, 14, 59, 120, 12
+    CONTROL "Se &obţine automat o adresă IP", IDC_USEDHCP, "BUTTON", BS_AUTORADIOBUTTON | WS_GROUP | WS_TABSTOP, 14, 43, 210, 12
+    CONTROL "Se &utilizează adresa IP următoare::", IDC_NODHCP, "BUTTON", BS_AUTORADIOBUTTON, 14, 59, 120, 12
     LTEXT "Adresă I&P:", -1, 14, 75, 135, 8
     CONTROL "", IDC_IPADDR, "SysIPAddress32", WS_TABSTOP, 144, 75 - 2, 85, 12
-    LTEXT "Mas&că de subrețea:", -1, 14, 90, 135, 8
+    LTEXT "&Mască subreţea:", -1, 14, 90, 135, 8
     CONTROL "", IDC_SUBNETMASK, "SysIPAddress32", WS_TABSTOP, 144, 90 - 2, 85, 12
-    LTEXT "S&erver „poartă” implicit:", -1, 14, 105, 135, 8
+    LTEXT "&Gateway implicit:", -1, 14, 105, 135, 8
     CONTROL "", IDC_DEFGATEWAY, "SysIPAddress32", WS_TABSTOP, 144, 105 - 2, 85, 12
     GROUPBOX "", -1, 9, 147, 228, 47, BS_GROUPBOX
-    CONTROL "Obține a&utomat adresa serverului DNS", IDC_AUTODNS, "BUTTON", BS_AUTORADIOBUTTON | WS_GROUP | WS_TABSTOP, 14, 129, 210, 12
-    CONTROL "Folosește adresa &DNS următoare", IDC_FIXEDDNS, "BUTTON", BS_AUTORADIOBUTTON, 14, 145, 152, 12
-    LTEXT "Server DNS p&referat:", -1, 14, 161, 135, 8
+    CONTROL "Se o&bţine automat adresa de server DNS", IDC_AUTODNS, "BUTTON", BS_AUTORADIOBUTTON | WS_GROUP | WS_TABSTOP, 14, 129, 210, 12
+    CONTROL "S&e utilizează adresa de server DNS următoare:", IDC_FIXEDDNS, "BUTTON", BS_AUTORADIOBUTTON, 14, 145, 152, 12
+    LTEXT "Server &DNS preferat::", -1, 14, 161, 135, 8
     CONTROL "", IDC_DNS1, "SysIPAddress32", WS_TABSTOP, 144, 161 - 2, 85, 12
     LTEXT "Server DNS &alternativ:", -1, 14, 176, 135, 8
     CONTROL "", IDC_DNS2, "SysIPAddress32", WS_TABSTOP, 144, 176 - 2, 85, 12
-    PUSHBUTTON "A&vansate", IDC_ADVANCED, 186, 199, 50, 14, WS_TABSTOP
+    PUSHBUTTON "Comple&x…", IDC_ADVANCED, 186, 199, 50, 14, WS_TABSTOP
 END
 
 IDD_TCPIP_ALTCF_DLG DIALOGEX 0, 0, 246, 216
 STYLE DS_SHELLFONT | WS_CHILD | WS_CAPTION
 CAPTION "Configurație alternativă"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "Dacă acest calculator este utilizat în mai multe rețele, puteți specifica mai jos o configurație alternativă.", -1, 9, 9, 220, 20
+    LTEXT "Dacă acest computer este utilizat în mai mult de o reţea, introduceţi setările IP alternative de mai jos.", -1, 9, 9, 220, 20
     GROUPBOX "", -1, 9, 57, 228, 125, BS_GROUPBOX
-    CONTROL "Adresă IP p&rivată obținută automat", IDC_USEDHCP, "BUTTON", BS_AUTORADIOBUTTON | WS_GROUP | WS_TABSTOP, 14, 40, 210, 12
-    CONTROL "Configurată &manual", IDC_NODHCP, "BUTTON", BS_AUTORADIOBUTTON, 14, 55, 90, 12
+    CONTROL "Ad&resă privată IP automată", IDC_USEDHCP, "BUTTON", BS_AUTORADIOBUTTON | WS_GROUP | WS_TABSTOP, 14, 40, 210, 12
+    CONTROL "&Configurat de utilizator", IDC_NODHCP, "BUTTON", BS_AUTORADIOBUTTON, 14, 55, 90, 12
```

Comment:
Please attach a pic-preview of the ros dlg for this line so that we can see we'll have no cutoff with the longer string and the 90 width.

Reply:
This is it:

![2024-05-09 22_42_11-Resource Hacker - netcfgx dll](https://github.com/reactos/reactos/assets/113553683/813426e5-691b-48c8-b838-c6118ce133cc)